### ISSUE TITLE
Crash when crafting with gunpowder

### ISSUE THREAD

**Issue by otakucode** [BevoTechPack/BevoTechPack#983]
So I noticed in NEI that there was a recipe that claimed if you crafted 9 gunpowder, you would get a vanilla "Fire" item.  I put 9 gunpowder in my ME Crafting Terminal and... crash.  Upon resume, the gunpowder is still in the ME Crafting Terminal, so opening it again results in another crash.  Haven't had a chance yet to try the recipe in a vanilla crafting table but I'm not terribly hopeful.  I didn't think you could ever acquire, let alone craft, Fire outside of being given the item by an admin.  I noticed there were recipes that could use the Fire to craft chain armor, and that the chain armor could be melted in a Tinker's smeltery to yield molten steel, so I thought maybe this was the 'surprise' way of getting steel hinted at in the books...  (if this is something I shouldn't mention here, let me know and I'll edit to remove)

**Comment by otakucode** [comment]
Just tested with the vanilla crafting table, the instant I place the 9th gunpowder in the crafting grid, the game crashes instantly.

**Comment by b0bst3r** [comment]
Yeah don't craft any of those recipes with fire in them.

**Comment by otakucode** [comment]
I certainly will not.  I also ran across something else, not sure if I should create a new issue for it or not... I'm not sure how interested in crashing bugs you are since development has moved on to 1.9.  But, I crafted the Forge Lexicon, and right-clicking it with it in my hand instacrashes the game.  I'm not familiar with the item, so I don't know what it actually does, but its tooltip says to right-click with it to study it or hit 'P' to gain wisdom.  Haven't tried hitting P.  Just socked it away in a chest after crashing out twice upon trying to open it with right-clicking on empty air.
